File: backend/Backend.DataAccess/DTO/Requests/UpdateUserRequest.cs
namespace Backend.API.DTO;

public record UpdateUserRequest(
    Guid Id,
    string? Email,
    string? PhoneNumber,
    string? Surname,
    string? Name,
    string? Patronymic,
    int? Age,
    string? Country,
    string? City,
    string? PhotoUrl
);

File: frontend/src/DTO/request/UpdateUserRequest.ts
import { UUID } from 'node:crypto';

export interface UpdateUserRequest {
    id?: string;
    // email?: string;
    // phoneNumber?: string;
    surname?: string;
    name?: string;
    patronymic: string | null;
    age?: number;
    country?: string;
    city?: string;
    photoUrl?: string;
}
